specification/network/resource-manager/Microsoft.Network/preview/2022-06-01-preview/networkVerifier.json

```diff
@@ -0,0 +1,1453 @@
+          }
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "ConfigurationSnapshotProperties": {
+      "required": [
+        "scopes"
+      ],
+      "type": "object",
+      "properties": {
+        "scopes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ConfigurationSnapshotScope"
+          },
+          "x-ms-identifiers": []
+        },
+        "createTime": {
+          "type": "string"
+        },
+        "errorMessage": {
+          "type": "string"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState"
+        }
+      }
+    },
+    "ConfigurationSnapshotScope": {
+      "type": "object",
+      "properties": {
+        "subscription": {
+          "type": "string"
+        }
+      }
+    },
+    "ErrorResponseBody": {
+      "type": "object",
+      "properties": {
+        "code": {
+          "type": "string"
+        },
+        "message": {
+          "type": "string"
+        },
+        "target": {
+          "type": "string"
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ErrorResponseBody"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "IPTrafficProperties": {
+      "type": "object",
+      "properties": {
+        "sourceIps": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "destinationIps": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "sourcePorts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "destinationPorts": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "protocols": {
+          "$ref": "#/definitions/NetworkProtocols"
+        }
+      }
+    },
+    "NetworkProtocols": {
+      "enum": [
+        "TCP",
+        "UDP",
+        "ICMP"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "NetworkProtocols"
+      }
+    },
+    "NetworkVerifier": {
+      "description": "Verifier instance to manage and run analysis.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true
+        },
+        "properties": {
+          "$ref": "#/definitions/NetworkVerifierProperties"
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-azure-resource": true,
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
```

Comment:
>/network.json#/definitions/Resource [](http://example.com/codeflow?start=20&length=35)

Use TrackedResource from v3 common types. Note, this will bring in systemData, which you should also support.